blackbox/docs/administration/auth_methods.txt

```diff
@@ -41,6 +41,15 @@ setting like this:
         http_default_user: dustin
 
 
+
+.. note::
+
+    When the Enterprise Edition, and the User Management are enabled,
+    the user of the Admin UI needs to be granted the following privileges:
+    ``DQL`` on ``sys.shards``, ``sys.nodes``, ``sys.node_checks``,
+    ``sys.checks``, ``sys.cluster``, and ``sys.jobs_log`` tables.
+    As well as ``DDL`` on the ``doc`` schema.
```

Comment:
I think you should elaborate why does the admin ui user need these privileges (it's not clear at all why it's important)